tests/endtoend/test_durable_functions.py

```diff
@@ -17,34 +15,23 @@
         or is_envvar_true(CONSUMPTION_DOCKER_TEST),
         "Docker tests cannot retrieve port needed for a webhook")
 class TestDurableFunctions(testutils.WebHostTestCase):
-
     @classmethod
-    def setUpClass(cls):
-        cls.env_variables['WEBSITE_HOSTNAME'] = "http:"
-        os_environ = os.environ.copy()
-        os_environ.update(cls.env_variables)

```

Comment:
we dont need to patch envs here?

Reply:
The patch env isn't resetting the env vars after the tests end. I think this is part of the problem for why the tests are flaky